src/main/view/KeyboardListener.cpp

```diff
@@ -178,6 +178,16 @@ void KeyboardListener::menuUpdate() {
 		previousState[sf::Keyboard::Return] = sf::Keyboard::isKeyPressed(sf::Keyboard::Return);
 	}
 
+	if (sf::Keyboard::isKeyPressed(sf::Keyboard::Escape) != previousState[sf::Keyboard::Escape]) {
+		if (sf::Keyboard::isKeyPressed(sf::Keyboard::Escape)) {
```

Comment:
We should probably refactor our keyboard stuff later to receive events from the main message loop.